db/migrate/20190206213630_add_icon_column_to_portfolioitem.rb

```diff
@@ -0,0 +1,5 @@
+class AddIconColumnToPortfolioitem < ActiveRecord::Migration[5.2]
+  def change
+    add_column :portfolio_items, :service_offering_icon_ref, :bigint
```

Comment:
@lindgrenj6 I was under impression we needed all ids to be strings. @mkanoor is that the case here as well?

Reply:
Unless it can't be due to what is delivered from Topology?

Reply:
Yep it should be a string, wrote this a long time ago when I didn't know that rule :\ fixed it!